lambdatimes/src/components/Content/Content.js

```diff
@@ -17,28 +17,36 @@ export default class Content extends Component {
   }
 
   componentDidMount() {
+    this.setState({
+      tabs:tabData,
+      cards:cardData,
+    })
+    
     // Once the component has mounted, get the data and reflect that data on the state.
   }
 
   changeSelected = tab => {
+    this.setState({
+      selected:tab
+    })
     // this function should take in the tab and update the state with the new tab.
   };
+   
 
   filterCards = () => {
-    /* Right now this function only returns the cards on state.
-      We're going to make this function more dynamic
-      by using it to filter out our cards for when a tab is selcted
-      
-      Notice that we're passing this function to our <Cards /> component below.
-      This function returns an array of cards, so we can just pass it down as such.
-
-      Your algorithim for the logic here is as follows: 
-        - if the selected tab is 'all' it should return all 
-          of the items from cardData. 
-        - else, it should only return those cards whose 'tab' matched this.state.selected.
-    */
-    return this.state.cards;
-  };
+   const cardArray = [...this.state.cards];
+   const newSelectedCardArray = [];
+   if(this.state.selected === 'all'){
+     return cardArray
+   }else{
+     cardArray.filter((card) =>{
```

Comment:
good job getting this to work, but this isn't the right way to use the `filter` method. It expects a returned true or false in the callback function